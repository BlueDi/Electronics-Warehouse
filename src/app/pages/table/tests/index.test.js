import React from 'react';
import renderer from 'react-test-renderer';
import WHTable from '../index';

test('Create a table', () => {
  const component = renderer.create(<WHTable match={{ params: { id: 2 } }} />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
