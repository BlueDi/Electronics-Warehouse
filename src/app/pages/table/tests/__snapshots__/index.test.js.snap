// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Create a table 1`] = `
<div
  className="ui active transition visible inverted dimmer"
  onClick={[Function]}
>
  <div
    className="content"
  >
    <div
      className="ui indeterminate inverted text loader"
    >
      Preparing Table
    </div>
  </div>
</div>
`;
